src/classes/Wildebeest.ts

```diff
@@ -422,7 +447,7 @@ export default class Wildebeest<TModels extends ModelMap> {
 
       this.logger.info('Waiting for the genesis migration to finish');
       // wait for some time
-      await randomTimeoutPromise(MIGRATION_TIMEOUT);
+      await sleepPromise(MIGRATION_TIMEOUT);
```

Comment:
we can remove `randomTimeoutPromise.ts` file.

Reply:
done